modules/xfem/doc/content/modules/xfem/index.md

```diff
@@ -1,6 +1,61 @@
 # XFEM Module
 
-!alert construction
-The XFEM module needs some documentation.
+!row!
+!col! small=12 medium=6 large=9
+The XFEM module implements the extended finite element method (XFEM) in the MOOSE framework. It can
+be used in conjuction with any number of other physics modules to enrich the solutions to partial
+differential equations (PDEs) with some form of discontinuity. XFEM theory and algorithms employed
+throughout this module are detailed in the following links:
 
-!syntax complete groups=XFEMApp
+- [Theory Manual](xfem/theory/theory.md)
+- [Algorithms](xfem/theory/algorithms.md)
+
+Tables provided in the [Objects, Actions, and Syntax](#objects-actions-and-syntax) section list
+objects contained within the XFEM module followed by a short explanation of the object's purpose.
+Links on the objects' names navigate to an individual detailed page for the object.
+
+!media xfem/image21.gif
+       style=width:100%;float:right;padding-top:2.7%;
+       caption=Crack propagation on a circular mesh as a result of hoop strain relief.
+
+!col-end!
+
+!col! small=12 medium=6 large=3
+!media xfem/image79.gif
+       caption=Side view of the deformation of a cylinder with a cut prescribed by XFEM.
+
+!media xfem/image78.gif
+       caption=Top view of the deformation of a cylinder with a cut prescribed by XFEM.
+!col-end!
+
+!row-end!
+
+## Applications
+
+XFEM is ideally suited to problems involving local discontinuities. Typically these types of
+problems fall into one of two categories: strong or weak discontinuities. Strong discontinuities
+are characterized by discontinuous solutions whereas a weak discontinuity is one involving a
+break in the flux at an interface. Examples of the types of problems XFEM can solve follow:
+
+- Cracking
+
+  - Prescribed crack growth
+  - Cracks propagating based on solution values (stress, pressure, etc.)
+
+- Moving Interfaces
+
+  - Dynamic material interfaces
+  - Open (glued) or closed (inclusion) interfaces on the domain
+  - Phase transition (Stefan) problems
+
+To support interfaces that move based upon calculated growth rates rather than prescribed functions,
```

Comment:
The locations of the interfaces used to define solution discontinuities can be prescribed in various ways, including by user-defined cutting planes and using level set functions. The evolution of level set functions can optionally be computed using the level set module (include link), which is recommended for interfaces that move based on calculated growth rates rather than prescribed functions.